src/app/components/videochat/jh-video-chat.directive.js

```diff
@@ -122,7 +122,11 @@
           }
           // If current one is still speaking, there is no need to change
           if ( !(current && current.getSpeaking()) ) {
```

Comment:
I wonder, shouldn't we change if `current` disables his/her video while speaking? I think we should. So I would change this to
```javascript
// If current one is still speaking and has not disabled the video, there is no need to change
if ( !(current && current.getSpeaking() && current.isEnabled("video") )) {
```
What do you think?